app/pages/external/TransitOperationDeclarationTypePage.scala

```diff
@@ -16,13 +16,10 @@
 
 package pages.external
 
-import models.DeclarationType
 import pages.{preTaskListPath, ReadOnlyPage}
 import play.api.libs.json.JsPath
 
-case object TransitOperationDeclarationTypePage extends ReadOnlyPage[DeclarationType] {
+case object TransitOperationDeclarationTypePage extends ReadOnlyPage[String] {
 
-  override def path: JsPath = preTaskListPath \ toString
-
-  override def toString: String = "declarationType"
+  override def path: JsPath = preTaskListPath \ "declarationType" \ "code"
```

Comment:
This is dependant on [3661](https://github.com/hmrc/ctc-departure-items-frontend/pull/193/files) being merged first

Reply:
It hopefully will be soon 🤞